routes/posts.js

```diff
@@ -0,0 +1,45 @@
+const db = require("/database/model.js");
+
+function get(request, response) {
+try {
+    const postsForm = /*html*/
+`
+    <form enctype="multipart/form-data class="Upload File" id="upload-file" action="/post" method="POST">
+    <div>
+      <label for="Title">Title:</label>
+      <input type="text" id="title" placeholder="Title" name="title" >
```

Comment:
id case should probably match label case (maybe the lowercase version preferable?)